pr_agent/algo/utils.py

```diff
@@ -316,19 +316,21 @@ def _fix_key_value(key: str, value: str):
     return key, value
 
 
-def load_yaml(response_text: str) -> dict:
+def load_yaml(response_text: str, keys_fix_yaml: List[str] = []) -> dict:
     response_text = response_text.removeprefix('```yaml').rstrip('`')
     try:
         data = yaml.safe_load(response_text)
     except Exception as e:
         get_logger().error(f"Failed to parse AI prediction: {e}")
-        data = try_fix_yaml(response_text)
+        data = try_fix_yaml(response_text, keys_fix_yaml=keys_fix_yaml)
     return data
 
-def try_fix_yaml(response_text: str) -> dict:
+
+def try_fix_yaml(response_text: str, keys_fix_yaml: List[str] = []) -> dict:
```

Comment:
**Suggestion:** In the `load_yaml` and `try_fix_yaml` functions, the default value for the `keys_fix_yaml` parameter is a mutable list. This can lead to unexpected behavior because the default list is shared between all calls. Consider changing the default value to `None` and initializing the list inside the function. [best practice]